src/Editor/GUI/Editors/ModelEditor.cpp

```diff
@@ -106,7 +106,7 @@ void ModelEditor::Show() {
                     // Create resource.
                     ResourceList::Resource resource;
                     resource.type = ResourceList::Resource::SCENE;
-                    resource.scene = model->name + "Scene";
+                    resource.scene = new std::string(model->name + "Scene");
```

Comment:
Memory leak since old scene string isn't deleted.
Instead just set the value.
*resource.scene = model->name + "Scene";